pkg/api/mlflow/config/config.go

```diff
@@ -19,6 +19,7 @@ type ServiceConfig struct {
 	AuthPassword          string
 	DefaultArtifactRoot   string
 	S3EndpointURI         string
+	GSEndpointURI         string
```

Comment:
I wonder if "GS" is too short to be clear? "S3" is okay because everyone knows it, but "GoogleStorage" might be clearer here. Or comment that "this is Google Storage"

Reply:
it looks consistent with `S3` and this parameter really needs only for integration tests. It is hidden even from user eyes